build.cmd

```diff
@@ -334,8 +343,22 @@ if '%TEST_PORTABLE_COREUNIT%' == '1' (
 )
 
 if '%BUILD_VS%' == '1' (
-    %_msbuildexe% %msbuildflags% VisualFSharp.sln /p:Configuration=%BUILD_CONFIG%
```

Comment:
What's the difference between VisualFSharp.sln and the other fours bits-and-pieces?  Are there more in one than the other in some way?